test/worlds/buoyancy_graded_restoring_moments.sdf

```diff
@@ -0,0 +1,79 @@
+<sdf version="1.6">
+  <world name="buoyancy">
+
+    <physics name="1ms" type="ode">
```

Comment:
```suggestion
    <physics name="1ms" type="ignored">
```

Nitpick. We've been removing `ode` from all these worlds because it can be misleading